randovania/games/samus_returns/json_data/Area 2 - Exterior.txt

```diff
@@ -399,12 +399,11 @@ Extra - asset_id: collision_camera_024
   * Tunnel to Outer Hub Shaft/Tunnel to Grapple Gap (Bottom)
   > Pickup (Aeion Tank)
       Any of the following:
-          Climb Rooms Vertically (No High Jump)
+          Simple IBJ
+          Morph Ball and Space Jump and Movement (Intermediate)
           Spiderspark (Beginner) and Can Spiderspark
   > Tunnel to Outer Hub Shaft (Top)
-      All of the following:
-          Lay Any Bomb
-          Single-wall Wall Jump (Beginner) or Climb Rooms Vertically (No High Jump)
+      Climb Rooms Vertically (No High Jump)
```

Comment:
I dont know the game as well, so i'll just ask: is it not possible to do this with a single walljump?
![grafik](https://github.com/randovania/randovania/assets/38186597/865d5412-8863-4511-b7f4-8c38d2e6baac)

Reply:
Right it's not part of the template. Ugh gotta fix that again